spec/Section 2 -- Language.md

```diff
@@ -587,13 +587,12 @@ FragmentName : Name but not `on`
 
 Fragments are the primary unit of composition in GraphQL.
 
-Fragments allow for the reuse of common repeated selections of fields, reducing
-duplicated text in the document. Inline Fragments can be used directly within a
-selection to condition upon a type condition when querying against an interface
-or union.
+Fragments allow for the definition of modular selection sets that make it easier
+to add and remove selections as needed.
```

Comment:
Maybe something along the lines of (written whilst on Zoom, not final wording!):

```
Fragments allow client components and logic to describe their specific
data requirements locally. Fragments can then be composed together to
form a GraphQL operation to issue to the server.
```